### PR TITLE
Added include_headers param in get_results method

### PR DESCRIPTION
**Pull request by tanishgupta1**
This param will add headers on results if its true.

### CODE REVIEW

File: bin/qds.py [hunk 1]
```diff
@@ -45,7 +45,8 @@
     "    run [cmd-specific-args .. ] : submit cmd & wait. print results\n"
     "    check <id> : print the cmd object for this id\n"
     "    cancel <id> : cancels the cmd with this id\n"
-    "    getresult <id> : get the results for the cmd with this id\n"
+    "    getresult <id> <enable_header>: id -> get the results for the cmd with this id\n"
+    "                                    enable_header -> to include headers in results(true/false)\n"
     "    getlog <id> : get the logs for the cmd with this id\n"
```
Comment (msumit) [inline bin/qds.py]: `enable_headers` doesn't sound correct.. u can keep it `include_headers` only..

File: bin/qds.py [hunk 2]
```diff
@@ -113,10 +114,10 @@ def submitaction(cmdclass, args):
         return 0
 
 
-def _getresult(cmdclass, cmd):
+def _getresult(cmdclass, cmd, args=[]):
```
Comment (msumit) [inline bin/qds.py]: use `kwargs` ie a hash instead of an array..

File: bin/qds.py [hunk 3]
```diff
@@ -156,9 +157,12 @@ def cancelaction(cmdclass, args):
 
 
 def getresultaction(cmdclass, args):
-    checkargs_id(args)
+    if len(args) > 2:
+        sys.stderr.write("expecting not more than 2 arguments\n")
+        usage()
+
```
Comment (msumit) [inline bin/qds.py]: no need of this check, once u make args a hash

File: qds_sdk/commands.py [hunk 4]
```diff
@@ -212,7 +212,11 @@ def get_results(self, fp=sys.stdout, inline=True, delim=None, fetch=True):
 
         conn = Qubole.agent()
 
-        r = conn.get(result_path, {'inline': inline})
+        enable_header = "false"
+        if len(arguments) == 1:
+            enable_header = arguments.pop(0)
```
Comment (msumit) [inline qds_sdk/commands.py]: no a good way to accept parameters.. make args a hash..

File: tests/test_command.py [hunk 5]
```diff
@@ -5,7 +5,7 @@
     import unittest
 else:
     import unittest2 as unittest
-from mock import Mock
+from mock import *
```
Comment (msumit) [inline tests/test_command.py]: why this change needed?

Comment (tanishgupta1) [inline tests/test_command.py]: call function was not declared which is present in mock , so included * to solve all dependencies. Same thing is done on test_actions.py to solve this.

File: bin/qds.py (hunk 2 above)
Comment (tanishgupta1) [inline bin/qds.py]: Looked into it.We can't pass hash here as its a single command line argument although it can be converted to hash internally but then we have create new function by converting single line arguments to hash . We generally use hash where we we use optparser or argumentparser and in return we get hash which is not the case here.

File: bin/qds.py (hunk 3 above)
Comment (tanishgupta1) [inline bin/qds.py]: same as above.

File: qds_sdk/commands.py (hunk 4 above)
Comment (tanishgupta1) [inline qds_sdk/commands.py]: same as above.

**Comment by msumit** [comment]
lgtm

File: qds_sdk/cluster.py [hunk 6]
```diff
@@ -737,6 +744,7 @@ def set_ec2_settings(self,
         self.ec2_settings['vpc_id'] = vpc_id
         self.ec2_settings['subnet_id'] = subnet_id
         self.ec2_settings['role_instance_profile'] = role_instance_profile
+        self.ec2_settings['master_elastic_ip'] = master_elastic_ip
```
Comment (harshshah87) [inline qds_sdk/cluster.py]: NIT: Can you add the setting description to the above Kwargs text.

File: qds_sdk/commands.py [hunk 7]
```diff
@@ -212,7 +212,11 @@ def get_results(self, fp=sys.stdout, inline=True, delim=None, fetch=True):
 
         conn = Qubole.agent()
 
-        r = conn.get(result_path, {'inline': inline})
+        include_header = "false"
+        if len(arguments) == 1:
+            include_header = arguments.pop(0)
```
Comment (harshshah87) [inline qds_sdk/commands.py]: Can we have a validation here= > include_header in ["true","false"]

File: bin/qds.py [hunk 8]
```diff
@@ -113,10 +114,10 @@ def submitaction(cmdclass, args):
         return 0
 
 
-def _getresult(cmdclass, cmd):
+def _getresult(cmdclass, cmd, args=[]):
     if Command.is_success(cmd.status):
         log.info("Fetching results for %s, Id: %s" % (cmdclass.__name__, cmd.id))
-        cmd.get_results(sys.stdout, delim='\t')
+        cmd.get_results(sys.stdout, delim='\t', qlog=cmd.qlog, arguments=args)
```
Comment (tanishgupta1) [inline bin/qds.py]: cmd is the object which is returned when we run the command. So qlog is one of the attribute of that object same like cmd.status in above condition.

File: qds_sdk/cluster.py (hunk 6 above)
Comment (tanishgupta1) [inline qds_sdk/cluster.py]: I think this is not my change. Have to merge latest unreleased branch to this.

File: qds_sdk/commands.py [hunk 9]
```diff
@@ -1065,6 +1076,8 @@ class DbImportCommand(Command):
                          help="Can be 1 for Hive export or 2 for HDFS/S3 export")
     optparser.add_option("--hive_table", dest="hive_table",
                          help="Mode 1: Name of the Hive Table from which data will be exported")
+    optparser.add_option("--hive_serde", dest="hive_serde",
```
Comment (tanishgupta1) [inline qds_sdk/commands.py]: same above this is not my change. Its due to merge issue.

File: qds_sdk/commands.py (hunk 7 above)
Comment (tanishgupta1) [inline qds_sdk/commands.py]: done
